drivers/sx127x/sx127x_netdev.c

```diff
@@ -54,6 +54,11 @@ static int _send(netdev_t *netdev, const iolist_t *iolist)
 
     uint8_t size = iolist_size(iolist);
 
+    /* Ignore send if packet size is 0 */
+    if (size == 0) {
+        return 0;
+    }
```

Comment:
Is this check really required?

Reply:
I added it just to avoid the SPI calls in case of empty packet

Reply:
ok.

Reply:
well to really avoid SPI calls, you should move this test even above the `_get_state` I guess - anyways this simple check should come first.